dependabot/resources/module_list.json

```diff
@@ -190,6 +190,13 @@
             "version_key": "graphqlVersion",
             "push_to_central": false
         },
+        {
```

Comment:
Shouldnt module-ballerina-peraist be added too?

Reply:
Added in [d295f35](https://github.com/ballerina-platform/ballerina-release/pull/2145/commits/d295f35bffabad650ba037ddc00f76458b246aac)